articles.tex

```diff
@@ -259,6 +269,19 @@
 \asubsection{Advisory Membership Term}
 Advisory Membership shall last until the member resigns.
 
+\asection{Leave of Absence}
+A leave of absence offers the option for members to take extended time away from their responsibilities to House for any number of personal issues including, but not limited to, physical illness, mental illness, or care giving for a sick family member. Computer Science House recognizes the need for its members to take care of their personal issues and will support them through the process of requesting a leave of absence as well as acclimating back to the culture of House upon their return. Upon approval of a leave of absence request, the Evaluations Director is notified and the member's leave begins immediately. A member is allowed to be physically present and also be on a leave of absence. The leave of absence can also be applied retroactively to the member at the discretion of the Evaluations Director.
+\asubsection{Leave of Absence Request}
+A leave of absence request must include the following information about the member: name, email, phone number, start date, expected return date, and a reason for the leave. The request is then given to a staff member of Residence Life, excluding student employees, for approval. At no point does the Executive Board need to be informed of any details relating to the reason for the leave.
+\asubsection{Leave of Absence Duration}
+A leave of absence can be granted for up to the length of one semester.
+\asubsection{Leave of Absence Extension}
+A leave of absence can be extended by submitting a new request.
+\asubsection{Leave of Absence Return}
+A member may return from a leave of absence whenever they choose to. When a member wishes to return, they must in writing notify a staff member of Residence Life, excluding student employees, to end their leave of absence.
```

Comment:
"They" is an appropriate gender neutral pronoun.
-- Matt
On Tue, Apr 26, 2016 at 9:33 PM Brandon Hudson <notifications@github.com>
wrote:

> In articles.tex
> <https://github.com/ComputerScienceHouse/Constitution/pull/56#discussion_r61192018>
> :
>
> > @@ -259,6 +269,19 @@
> >  \asubsection{Advisory Membership Term}
> >  Advisory Membership shall last until the member resigns.
> >
> > +\asection{Leave of Absence}
> > +A leave of absence offers the option for members to take extended time away from their responsibilities to House for any number of personal issues including, but not limited to, physical illness, mental illness, or care giving for a sick family member. Computer Science House recognizes the need for its members to take care of their personal issues and will support them through the process of requesting a leave of absence as well as acclimating back to the culture of House upon their return. Upon approval of a leave of absence request, the Evaluations Director is notified and the member's leave begins immediately. A member is allowed to be physically present and also be on a leave of absence. The leave of absence can also be applied retroactively to the member at the discretion of the Evaluations Director.
> > +\asubsection{Leave of Absence Request}
> > +A leave of absence request must include the following information about the member: name, email, phone number, start date, expected return date, and a reason for the leave. The request is then given to a staff member of Residence Life, excluding student employees, for approval. At no point does the Executive Board need to be informed of any details relating to the reason for the leave.
> > +\asubsection{Leave of Absence Duration}
> > +A leave of absence can be granted for up to the length of one semester.
> > +\asubsection{Leave of Absence Extension}
> > +A leave of absence can be extended by submitting a new request.
> > +\asubsection{Leave of Absence Return}
> > +A member may return from a leave of absence whenever they choose to. When a member wishes to return, they must in writing notify a staff member of Residence Life, excluding student employees, to end their leave of absence.
>
> Possible change for "they" to "he/she" where applicable. Also change tense
> for verbs as needed.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/ComputerScienceHouse/Constitution/pull/56/files/2b48362af7d87a9f67f2c563673b6b8567ed4414#r61192018>
>